tests/cases/tmmock2tmmock/configs/demo/ibc-0.json

```diff
@@ -0,0 +1,15 @@
+{
+  "chain": {
+    "@type": "/relayer.chains.tendermint.config.ChainConfig",
+    "key": "testkey",
+    "chain_id": "ibc0",
+    "rpc_addr": "http://localhost:26657",
+    "account_prefix": "cosmos",
+    "gas_adjustment": 1.5,
+    "gas_prices": "0.025stake"
+  },
+  "prover": {
+    "@type": "/relayer.provers.mock.config.ProverConfig",
+    "finality_delay": 5
```

Comment:
@dongrie This long delay makes CI slow. This was my bad choice. Could you change this to `3`?
Please do the same for `ibc-1.json`.

Reply:
https://github.com/hyperledger-labs/yui-relayer/pull/96/commits/498008702023ac2eef6de6a83f96c5f511e047d3
Fixed